public/locales/pt/common.json

```
@@ -0,0 +1,15 @@
+{
+  "title": "O modo mais fácil de criar um",
+  "readme": "README",
+  "description": "Nosso simples editor permite que você facilmente adicione e customize todas as seções que você precisa para o readme do seu projeto",
```

We don't have to use the anglicism here :)

```suggestion
  "description": "O nosso simples editor permite que você facilmente adicione e personalize todas as seções que você precisa para o README do seu projeto",
```

---

I don't think anglicism is a problem, and the `customize` sounds awkward in the European Portuguese?
I agree with the `README`

---

It's not awkward, just unnecessary. Out of curiosity, does "personalizar" sound odd to you, compared to "customizar"? Or is it just irrelevant?

---

They're irrelevant, the two words have the same meaning haha
We can go with your suggestion :)